app/Services/Model/Imp/SummitService.php

```diff
@@ -2028,6 +2028,52 @@ public function removeModeratorFromPresentation(int $current_member_id, int $spe
         });
     }
 
+    /**
+     * @param int $current_member_id
+     * @param int $speaker_id
+     * @param int $presentation_id
+     * @param int $new_order
+     * @return Presentation
+     * @throws EntityNotFoundException
+     * @throws ValidationException
+     */
+    public function updateSpeakerOrderInPresentation(int $current_member_id, int $speaker_id, int $presentation_id, int $new_order): Presentation {
+        return $this->tx_service->transaction(function () use ($current_member_id, $speaker_id, $presentation_id, $new_order) {
+
+            $current_member = $this->member_repository->getById($current_member_id);
```

Comment:
this method is a admin endpoint
u dont need to check the authz stuff
should be only checked by middleware

Reply:
done